odl/tomo/backends/astra_setup.py

```diff
@@ -64,16 +63,50 @@
 from odl.tomo.geometry import (
     Geometry, Parallel2dGeometry, DivergentBeamGeometry, ParallelGeometry,
     FlatDetector)
+from odl.util.utility import pkg_supports
 
 
-__all__ = ('ASTRA_AVAILABLE', 'ASTRA_VERSION',
+__all__ = ('ASTRA_AVAILABLE', 'ASTRA_VERSION', 'astra_supports',
            'astra_volume_geometry', 'astra_projection_geometry',
            'astra_data', 'astra_projector', 'astra_algorithm',
            'astra_conebeam_3d_geom_to_vec',
            'astra_conebeam_2d_geom_to_vec',
            'astra_parallel_3d_geom_to_vec')
 
 
+# Dictionary specifying which ASTRA versions support which feature. The dict
+# values must be specified as valid setuptools package requirements without
+# package name, e.g., as exact version '==1.7', as inequality '>=1.7' or as
+# several requirements that need to be satisfied simultaneously, e.g.,
+# '>=1.8,<=2.0'.
+# To give multiple requirements that should be OR-ed together, use a
+# sequence instead of a single string in the dictionary, e.g.,
+# ['==1.7', '>=1.8,<=2.0'].
+ASTRA_FEATURES = {'anisotropic voxels 2d': None,
```

Comment:
Really weird, I had the same feeling while changing it ... I'll revert that change.